src/objects/featured-row/featured-row.scss

```diff
@@ -0,0 +1,47 @@
+@use "../../compiled/tokens/scss/breakpoint";
+@use "../../compiled/tokens/scss/size";
+@use '../../mixins/fluid';
+@use '../../mixins/media-query';
+@use "../../mixins/ms";
+
+.o-featured-row {
+  display: grid;
+  @include fluid.grid-gap(
+    breakpoint.$s,
+    breakpoint.$xl,
+    size.$spacing-gap-fluid-min,
+    size.$spacing-gap-fluid-max
+  );
+  grid-template-areas:
+    'intro'
+    'content'
+    'action';
+  grid-template-rows: repeat(3, minmax(0, auto));
+
+  @media (min-width: breakpoint.$l) {
+    grid-template-areas:
+      'intro content content'
+      '. content content'
+      'action content content';
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: minmax(0, auto) 1fr minmax(0, auto);
+  }
+}
+
+.o-featured-row__intro {
+  @media (min-width: breakpoint.$l) {
+    grid-area: intro;
+  }
+}
+
+.o-featured-row__content {
+  @media (min-width: breakpoint.$l) {
+    grid-area: content;
+  }
+}
+
+.o-featured-row__action {
+  @media (min-width: breakpoint.$l) {
+    grid-area: action;
+  }
```

Comment:
I believe the media queries are no longer necessary since the `grid-area` name for these does not change?